python/whylogs/api/writer/whylabs.py

```diff
@@ -111,8 +111,9 @@ def _check_whylabs_condition_count_uncompound() -> bool:
             return False
         else:
             logger.info(f"Got response code {response.status_code} but expected 200, so running uncompound")
-    except Exception:
-        logger.warning("Error trying to read whylabs config, falling back to defaults for uncompounding")
+    finally:
+        pass
```

Comment:
Instead of this, can you change the verbosity of this log statement from warning to info/debug?

You need to add back the:
```python
except Exception: 
```

A `finally` block will attempt to run after an exception but your updated code no longer handles the exception. We don't want this operation to block writing profiles if they can't read this value, we can fallback to the default.